lightning/src/ln/channelmanager.rs

```diff
@@ -6994,7 +7142,7 @@ where
 				if peer_state_mutex_opt.is_none() { return; }
 				let mut peer_state_lock = peer_state_mutex_opt.unwrap().lock().unwrap();
 				let peer_state = &mut *peer_state_lock;
-				if let Some(chan) = peer_state.channel_by_id.get_mut(&msg.channel_id) {
+				if let Some(chan) = peer_state.outbound_v1_channel_by_id.get_mut(&msg.channel_id) {
```

Comment:
Oh wait nevermind. Got you. The outbound still makes sense here. I've collected all the keys in the above case instead of just `channel_by_id`'s keys.